server/models/Transaction.js

```diff
@@ -435,6 +435,20 @@ export default (Sequelize, DataTypes) => {
     return exportToCSV(transactions, attributes, getColumnName, processValue);
   };
 
+  Transaction.createPlatformTipTransaction = async transaction => {
```

Comment:
I think we should keep logic within the `processOrder` function.
My reasoning here is that this special condition only applies to funds for now and this function can cause some confusion with the `feesOnTop` one.

Reply:
@kewitz : Thanks. Done, let me know if you see any further issues. 😄